addons/crm/views/crm_lead_views.xml

```diff
@@ -591,6 +591,58 @@
             </field>
         </record>
 
+        <record id="crm_lead_view_kanban_forecast" model="ir.ui.view">
```

Comment:
You should probably give priorities to your views to ensure they are never chosen as default views (default views generally have 10 or 16 I think, you could set a priority to 32 to set your views as "less prior").